CHANGELOG.md

```diff
@@ -46,9 +71,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - Update pytest-cov requirement from ~=4.0.0 to ~=4.1.0 ([#697](https://github.com/awslabs/syne-tune/issues/697))
 - enable caching for unit tests, fix README typo ([#695](https://github.com/awslabs/syne-tune/issues/695))
 
+[v0.8.0]: https://github.com/awslabs/syne-tune/compare/v0.7.0...v0.8.0
```

Comment:
Nit: link already exists on line 34, but having it duplicated should not impact how the page renders